tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -47,15 +47,18 @@ def setUp(self):
         self._empty_bucket()
         test_util.delete_all_tables(self.bigquery_dataset_id)
 
-
-#        self._create_drug_class_table()
+        # self.create_drug_class_table()
```

Comment:
Does this need to be uncommented to create the drug class table?

Reply:
This was commented out already when I checked out the develop. I noticed this had been resolved in the develop. Will merge it